test/unit/model/scope.test.ts

```diff
@@ -102,13 +102,32 @@ describe('src/model/scope', (): void => {
       expect(scopes[0]).toEqual(expect.objectContaining(tempScope))
     })
 
+    it('adds multiple scopes for an existing consent to the database', async (): Promise<void> => {
```

Comment:
in the test above `adds scope for an existing consent to the database` you should test that only one scope has been added